homeassistant/components/rainbird/quality_scale.yaml

```diff
@@ -0,0 +1,58 @@
+rules:
+  # Bronze
+  config-flow: done
```

Comment:
The data_description of the password field is missing in the configuration flow.